drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/columns/EnumSingleSelectDateUiColumn.java

```diff
@@ -0,0 +1,106 @@
+
+    @Override
+    protected void initialiseMultiValueDomElement( final IGridCell<Date> cell,
+                                                   final GridBodyCellRenderContext context,
+                                                   final Map<String, String> enumLookups ) {
+        factory.attachDomElement( context,
+                                  new Callback<ListBoxDOMElement<Date, ListBox>>() {
+                                      @Override
+                                      public void callback( final ListBoxDOMElement<Date, ListBox> e ) {
+                                          final ListBox widget = e.getWidget();
+                                          for ( Map.Entry<String, String> lookup : enumLookups.entrySet() ) {
+                                              widget.addItem( lookup.getValue(),
+                                                              lookup.getKey() );
+                                          }
+                                          factory.toWidget( cell,
+                                                            widget );
+                                      }
+                                  },
+                                  new Callback<ListBoxDOMElement<Date, ListBox>>() {
+                                      @Override
+                                      public void callback( final ListBoxDOMElement<Date, ListBox> e ) {
+                                          e.getWidget().setFocus( true );
+                                      }
+                                  } );
+    }
+
+    @Override
+    protected void initialiseSingleValueDomElement( final IGridCell<Date> cell,
+                                                    final GridBodyCellRenderContext context ) {
+        singleValueFactory.attachDomElement( context,
+                                             new Callback<DatePickerDOMElement>() {
+                                                 @Override
+                                                 public void callback( final DatePickerDOMElement e ) {
+                                                     final DatePicker widget = e.getWidget();
+                                                     if ( hasValue( cell ) ) {
+                                                         widget.setValue( cell.getValue().getValue() );
+                                                     } else {
+                                                         widget.setValue( new Date() );
+                                                     }
+                                                 }
+                                             },
+                                             new Callback<DatePickerDOMElement>() {
+                                                 @Override
+                                                 public void callback( final DatePickerDOMElement e ) {
+                                                     //Do nothing
```

Comment:
There is usually focus setting code here (in other column impls, even another Date related column). Excuse the silly question, but why not here as well?